diff --git a/packages/injected/src/consoleApi.ts b/packages/injected/src/consoleApi.ts
index 2f2da420b..1f5131c26 100644
--- a/packages/injected/src/consoleApi.ts
+++ b/packages/injected/src/consoleApi.ts
@@ -71,7 +71,7 @@ declare global {
   interface Window {
     playwright?: any;
     inspect: (element: Element | undefined) => void;
-    __pw_resume?: () => Promise<void>;
+    __nx_resume?: () => Promise<void>;
   }
 }
 
@@ -139,8 +139,8 @@ export class ConsoleAPI {
   }
 
   private _resume() {
-    if (!this._injectedScript.window.__pw_resume)
+    if (!this._injectedScript.window.__nx_resume)
       return false;
-    this._injectedScript.window.__pw_resume().catch(() => {});
+    this._injectedScript.window.__nx_resume().catch(() => {});
   }
 }
